Update troubleshooting.mdx (#767)
helium/docs: docs/use-the-network/console/troubleshooting.mdx

- It takes up to (20-30 mins) for a sensor to be added for the first time due to XOR filter adding sensor credentials to blockchain: for more info check [here](https://docs.helium.com/use-the-network/console/adding-devices/#when-adding-devices-for-the-first-time).

### Why aren't downlinks working?

> LoRaWAN class A devices require an uplink before they can downlink. According to the LoRaWAN spec a receive window exists for the device to "hear" the downlink. 

Reviewer: ```suggestion
LoRaWAN class A devices require an uplink before they can downlink. According to the LoRaWAN spec, a receive window exists for the device to "hear" the downlink. 
```

> If this window (default 1 second) is missed then the downlink will be unsuccessful. With the Receive Window Delay feature users can 

Reviewer: ```suggestion
If this window (which defaults to `1` second) is missed then the downlink will be unsuccessful. With the `Receive Window Delay` feature users can 
```

increase the window length and address latency issues that can prevent downlinks. More information about how to use profiles to configure the 

> receive window delay for devices and groups [here](https://docs.helium.com/use-the-network/console/profiles/#receive-window-delay).

Reviewer: ```suggestion
receive window delay for devices and groups [can be found here](https://docs.helium.com/use-the-network/console/profiles/#receive-window-delay).
```
